src/components/top-features.js

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+
+ const TopFeatures = () => (
+  <section>
+    <div class="cmp-percentage-card-grid">
+      <div class="cmp-percentage-card">
+        <svg class="cmp-percentage-card__icon" width="264" height="264" viewBox="0 0 264 264">
```

Comment:
If you wouldn’t mind looking into it, I’d love for the colors of the SVGs to be variables. I’m not sure if React does global variables, but that would definitely be nice. That way we can maintain color control without having a bunch of hex values floating around.

Reply:
I spent a good hour looking for options for this and couldn't find anything. If you have more direction to provide, I could time box more time for it, but I'm not sure what else to try/search.